extensions/common/xwalk_external_extension.h

```diff
@@ -39,8 +40,8 @@ class XWalkExternalExtension : public XWalkExtension {
 
   bool Initialize();
 
-  void set_runtime_variables(const base::ValueMap& runtime_variables) {
-    runtime_variables_ = runtime_variables;
+  void set_runtime_variables(base::DictionaryValue::Storage* runtime_variables) {
+      runtime_variables_.swap(*runtime_variables);
```

Comment:
Logically it is ok to use swap but I don't think we need to make setter function that complicated. Wouldn't simply assigning the value or using std::move work here?